browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -9506,7 +9508,7 @@ index e4e88a11311aba49c65e6bc84767a1188588db36..33136b2448281b26c8c441626e38e536
 +{
 +    ASSERT(RunLoop::isMain());
 +
-+    m_queue->dispatch([this, protectedThis = makeRef(*this), sessionID, completionHandler = WTFMove(completionHandler)]() mutable {
++    m_queue->dispatch([this, protectedThis = Ref { *this }, sessionID, completionHandler = WTFMove(completionHandler)]() mutable {
```

Comment:
Why is this changing? Was `makeRef` removed from WebKit?

Reply:
Right, `makeRef` and `makeRefPtr` were removed from WebKit codebase recently:
  - 2021-09-22 Drop makeRef() and use Ref { } instead https://bugs.webkit.org/show_bug.cgi?id=230626
  - 2021-09-22 Reduce use of makeRef() and use Ref { } instead https://bugs.webkit.org/show_bug.cgi?id=230585
  - 2021-09-21 Drop makeRefPtr() and use RefPtr { } directly instead https://bugs.webkit.org/show_bug.cgi?id=230564
  - 2021-09-21 Drop remaining uses of makeRefPtr() in WebCore/ https://bugs.webkit.org/show_bug.cgi?id=230527
  - 2021-09-20 Reduce use of makeRefPtr() and use RefPtr { } directly https://bugs.webkit.org/show_bug.cgi?id=230503
  - 2021-09-20 Stop using makeRef(*this) / makeRefPtr(this) https://bugs.webkit.org/show_bug.cgi?id=230464